doc/source/getting_started/develop_pymapdl.rst

```diff
@@ -0,0 +1,337 @@
+
+.. _developing_pymapdl:
+
+===============
+Develop PyMAPDL
+===============
+
+You can help PyMAPDL library by fixing an bug or developing a new feature.
+To do that, you need to setup the repository in your local machine as it is explained
+in the following sections.
+
+
+Cloning the PyMAPDL repository
+==============================
+
+To clone PyMAPDL repository, you'll need to install a version control system (VCS) such as Git.
+Later run this code to clone the latest development version of PyMAPDL:
+
+.. code:: console
+
+    git clone https://github.com/ansys/pymapdl
+    cd pymapdl
+
+
+
+Create a Python virtual environment
+===================================
+
+It is recommended you install PyMAPDL in its own virtual environment to avoid
+dependency conflicts and to easier manage upgrades.
+Visit `Setting up your development environment <dev_guide_setup_your_environment_>`_
+for detailed information on how to install Python and create a virtual environment.
+
+Install PyMAPDL in development mode
+===================================
+
+Install the latest version of PyMAPDL in development mode:
+
+
+.. code:: console
+
+    cd pymapdl
+    pip install pip -U
+    pip install -e .
+
+
+If you are going to do testing, you need to install the testing dependencies:
+
+
+.. code:: console
+
+    pip install -e '.[tests]'
+
+
+Develop PyMAPDL
```

Comment:
Changing the earlier title to "Develop code" at least makes this title unique, which is something you want.

Reply:
> The following section is great

thank you! :)

Well... I can do a PR on that guide... and copy this over there.... In the meantime, we can keep this here. :)

Reply:
Pinging @jorgepiloto